.github/workflows/build.yml

```diff
@@ -114,7 +114,7 @@ jobs:
       run: |
         mkdir build
         cd build
-        cmake .. -DCMAKE_BUILD_TYPE=Release -DCMAKE_C_COMPILER_LAUNCHER=ccache -DCMAKE_CXX_COMPILER_LAUNCHER=ccache
+        cmake .. -DCMAKE_BUILD_TYPE=Release -DCMAKE_C_COMPILER_LAUNCHER=ccache -DCMAKE_CXX_COMPILER_LAUNCHER=ccache -DWICKED_USE_IPO=NO
```

Comment:
why also for non-pr?

Reply:
we need it for the master branch because PRs cannot share caches with each other, they can only take the cache from master

Reply:
I usually take the latest build artifact when updating releases because that can be more up to date than the nightly. Why do prs have to share cache between each other? Within one PR would the ccache work if the not-pr still use the link time optimization?